js/app.js

```diff
@@ -0,0 +1,79 @@
+/*jshint esversion: 6 */
+let person4 = new XMLHttpRequest();
+function name4Person() {
+  let obj = JSON.parse(this.responseText);
+  document.getElementById('person4Name').innerHTML = obj.name;
+
+  let planet1 = new XMLHttpRequest();
+  planet1.open('GET', obj.homeworld);
+  planet1.addEventListener('load',function() {
+    let homeWorld = JSON.parse(this.responseText);
+    document.getElementById('person4HomeWorld').innerHTML = homeWorld.name;
+  });
+  planet1.send();
+}
+person4.addEventListener('load', name4Person);
+person4.open("GET", "http://swapi.co/api/people/4/");
+person4.send();
+
+var person14Name = new XMLHttpRequest();
+function name14Person() {
+  let person = JSON.parse(this.responseText);
+  document.getElementById('person14Name').innerHTML = person.name;
+
+  let species = new XMLHttpRequest();
+  species.open("GET", person.species);
```

Comment:
So this only works because the array only contained one string. If there was more than one item in the array, it would have joined them all together. This could become a bug in the future so it's best to refer to just the species you wanted to list.